### ISSUE TITLE
Improving landing to the site

### ISSUE THREAD

**Issue by EnzoVezzaro** [oceanprotocol/df-issues#169]

**Comment by jamiehewitt15** [comment]
Shouldn't these issues and this epic be in the df-web repository?

**Comment by EnzoVezzaro** [comment]
you're right. I used zenhub to create them, but I though I were on the correct repo. 

@idiom-bytes let's talk about it during our section today.
